pkgs/applications/video/animdl/default.nix

```
@@ -0,0 +1,47 @@
+{ python3Packages, buildPythonApplication, fetchFromGitHub, lib,
+  poetry-core,
+  anchor-kr,
+  anitopy,
+  click,
+  cssselect,
+  httpx,
+  lxml,
+  packaging,
+  pkginfo,
+  pycryptodomex,
+  pyyaml,
+  regex,
+  rich,
+  tqdm,
+  yarl
+}:
+buildPythonApplication rec {
+  pname = "animdl";
+  version = "1.7.17";
+  src = fetchFromGitHub {
+    owner = "justfoolingaround";
+    repo = "${pname}";
+    # Using the commit hash because upstream does not have releases. https://github.com/justfoolingaround/animdl/issues/277
+    rev = "fd3277fb919c4b5e7ef42daba0bd001695a9c124";
+    hash = "sha256-pgtrStorkIJJ3oIGgRjNWYwcQ5jUC8IS5iI9CyDJkzE=";
+  };
+  format = "pyproject";
+  nativeBuildInputs = [ poetry-core cssselect ];
+  propagatedBuildInputs = [ anchor-kr anitopy click httpx lxml packaging pkginfo pyyaml pycryptodomex regex rich tqdm yarl ];
```

```suggestion
  version = "1.7.17";
  format = "pyproject";

  src = fetchFromGitHub {
    owner = "justfoolingaround";
    repo = "${pname}";
    # Using the commit hash because upstream does not have releases. https://github.com/justfoolingaround/animdl/issues/277
    rev = "fd3277fb919c4b5e7ef42daba0bd001695a9c124";
    hash = "sha256-pgtrStorkIJJ3oIGgRjNWYwcQ5jUC8IS5iI9CyDJkzE=";
  };

  nativeBuildInputs = [
    poetry-core
    cssselect
  ];

  propagatedBuildInputs = [
    anchor-kr
    anitopy
    click
    httpx
    lxml
    packaging
    pkginfo
    pyyaml
    pycryptodomex
    regex
    rich
    tqdm
    yarl
  ];
```

just some formatting nitpicks